analyzer/runtime/runtime.go

```diff
@@ -65,16 +65,16 @@ func NewRuntimeAnalyzer(
 	return block.NewAnalyzer(blockRange, batchSize, mode, string(runtime), processor, target, logger)
 }
 
-func (m *processor) nativeTokenSymbol() string {
-	return m.runtimeMetadata.Denominations[sdkConfig.NativeDenominationKey].Symbol
+func nativeTokenSymbol(runtimeMetadata *sdkConfig.ParaTime) string {
+	return runtimeMetadata.Denominations[sdkConfig.NativeDenominationKey].Symbol
 }
 
 // StringifyDenomination returns a string representation of the given denomination.
 // This is simply the denomination's symbol; notably, for the native denomination,
 // this is looked up from network config.
-func (m *processor) StringifyDenomination(d sdkTypes.Denomination) string {
+func StringifyDenomination(runtimeMetadata *sdkConfig.ParaTime, d sdkTypes.Denomination) string {
```

Comment:
The alternative to this refactor would be to copy these methods to `extract.go`, which would leave us with duplicate `processor.StringifyDenomination` as well as a `StringifyDenomination` functions in the same package. Not sure which is cleaner/more idiomatic, open to suggestions.